server/liveblog/posts/posts.py

```diff
@@ -158,9 +162,10 @@ def on_updated(self, updates, original):
         # send notifications
         if updates.get('deleted', False):
             push_notification('posts', deleted=True, post_id=original.get('_id'))
-        elif updates.get('post_status') == 'draft':
-            push_notification('posts', drafted=True, post_id=original.get('_id'),
-                              author_id=original.get('original_creator'))
+        # NOTE: Seems unsused, to be removed later if no bug appears.
```

Comment:
I commented it to let us able to use it later, when we will need it. For now this is useless, and incomplete.